hack/istio/multicluster/kiali-prepare-remote-cluster.sh

```diff
@@ -456,6 +481,11 @@ Valid command line arguments:
   -vo|--view-only: if 'true' then the created service account/remote secret
                    will only provide a read-only view of the remote cluster.
                    Default: "${DEFAULT_VIEW_ONLY}"
+  -ea|--exec-auth: If 'true', use the mocked exec auth for remote secret authentication.
```

Comment:
this hack script is going to be used (documented as a helper script) by people that will be using the new multi-cluster features in Kiali (a blog post will be available shortly introducing this stuff). So even though this is in a "hack" directory, it really is going to be used by people other than just kiali devs (think of this as the poor man's "istioctl create-remote-secret").

So, let's not put "mock" things in here. Let's change this option to something like this (unless you can think of a better way).... Change it to `--exec-auth-json` which takes a file path - that file contains the json you want to inject into the secret's kubeconfig. This way, people can use this "for real" to inject their actual config.

Reply:
Removed mock from hack script and added --exec-auth-json option instead.
Slightly different from the comments, the --exec-auth-json option does not require a file path, but an actual json payload.
This allows more flexibility when using scripts.
If you want to use file paths, you can use scripts like this.
`./kiali-prepare-remote-cluster.sh -eaj "$(cat . /exec-auth.json)"`